kmip/services/server/engine.py

```diff
@@ -671,16 +671,8 @@ def _process_create(self, payload):
             )
 
         length = object_attributes.get('Cryptographic Length')
-        if length:
+        if length is not None:
             length = length.value
-        else:
-            # TODO (peterhamilton) The cryptographic length is technically not
-            # required per the spec. Update the CryptographyEngine to accept a
-            # None length, allowing it to pick the length dynamically. Default
-            # to the strongest key size allowed for the algorithm type.
```

Comment:
Now that I think about it, the cryptography engine's `create_symmetric_key` method will need to also include the `key_length` in the result dict, since the length is needed for storing the whole managed object (line 699 below).